proto/regen/ecocredit/basket/v1/state.proto

```diff
@@ -23,17 +24,22 @@ message Basket {
     // basket_denom is the basket bank denom.
     string basket_denom = 2;
 
+    // name is the unique name of the basket specified in MsgCreate. Basket
+    // names must be unique across all credit types and choices of exponent
+    // above and beyond the uniqueness constraint on basket_denom.
+    string name = 3;
```

Comment:
can we automatically create an "empty" index? Essentially a set of keys, without associating any structure with it through ORM. Otherwise maybe it will make more sense to do it "manually"?

Reply:
what do you mean by empty?

Reply:
if the only thing we want is a set of names, then we would do it by mapping `name -> 1`, without adding the name to the struct and doing `name -> primary key` association.

Reply:
I'm using 1 since we can't use empty value. In go it would be `name -> empty struct`

Reply:
we can use an empty array, but the orm has this all covered still